src/annotator/cross-frame.js

```diff
@@ -31,10 +31,10 @@ export class CrossFrame {
    */
   constructor(element, eventBus, options) {
     const { config, server } = options;
-    const discovery = new Discovery(window, { server });
-    const bridge = new Bridge();
-    const annotationSync = new AnnotationSync(eventBus, bridge);
-    const frameObserver = new FrameObserver(element);
+    this.discovery = new Discovery(window, { server });
```

Comment:
Should these fields be private? If so they should use an underscore prefix per our conventions. When designing the API for a class I recommend to default to making fields private and reducing the API surface area unless you know of a good use case for exposing the field.

Reply:
I prepended all internal fields with underscore on a follow up commit.

I suggest you merge first your pending PR https://github.com/hypothesis/client/pull/3611 and then I can resolve the conflicts.